src/apis/platformvm/builder.ts

```diff
@@ -1831,19 +1835,20 @@ export class Builder {
     if (this._feeCheck(fee, feeAssetID)) {
       const aad: AssetAmountDestination = new AssetAmountDestination(
         [],
-        0,
+        toThreshold,
         fromSigner.from,
         fromSigner.signer,
         changeAddresses,
         changeThreshold
       )
-      aad.addAssetAmount(feeAssetID, zero, fee)
+      //aad.addAssetAmount(feeAssetID, zero, fee)
+      aad.addAssetAmount(stakeAssetID, stakeAmount, fee) // TODO: @VjeraTurk Is it ok like so?
```

Comment:
@aeddaqqa @evlekht This works for the examples, 
but I'm unsure how to check if it will also work for camino network

Reply:
yes, looks correct
you should test it with kopernikus network
if it builds tx and node accepts it, than its fine